### PR TITLE
Correct PostgreSQL installation instructions for macOS - Closes #348

### LINKED ISSUE #348: PostgreSQL installation instructions for macOS are erroneous
We should not recommend setting [LDFLAGS and CPPFLAGS](https://github.com/LiskHQ/lisk-docs/blame/75f9ddf978ccc9836148760fce401956b73c0158/lisk-core/setup/source.md#L214)

`initdb /usr/local/var/postgres -E utf8 --locale=en_US.UTF-8` 
- initializes `/usr/local/var/postgres` while `/usr/local/var/postgresql@10` gets used
- `/usr/local/var/postgresql@10` has already been initialized by brew

I suspect `--createdb` is superfluous since we go on to create a use a `lisk_<NETWORK>` database.
```
createuser --createdb lisk
createdb lisk_<NETWORK> --owner lisk
```

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: lisk-core/setup/source.md [hunk 1]
```diff
@@ -208,24 +208,22 @@ Install Postgres version 10:
...
 Start Postgres, create the `lisk` user and the database:
 ```bash
-initdb /usr/local/var/postgres -E utf8 --locale=en_US.UTF-8
-brew services start postgresql@10
-createuser --createdb lisk
-createdb lisk_<NETWORK> --owner lisk
-psql -d lisk_<NETWORK> -c "alter user lisk with password 'password';"
+pg_ctl -D /usr/local/var/postgresql@10 start
+createuser lisk
+createdb --owner=lisk isk_<NETWORK>
```
Comment (Nazgolze) [inline lisk-core/setup/source.md]:
```suggestion
createdb --owner=lisk lisk_<NETWORK>
```

**Review by Nazgolze — CHANGES_REQUESTED** [review]
minor typo

**Review by Nazgolze — APPROVED** [review]

**Review by Tschakki — CHANGES_REQUESTED** [review]
Hey!
The `development` branch is not used at the moment, please choose `docs-sdk` and `docs-core` as base branches for the fix. It needs to be fixed in core source setup as well as in the sdk setup, thanks :)

**Comment by ghost** [comment]
@Tschakki Isn't `development` what's on https://lisk.io/documentation/lisk-core/setup at the moment? (and `docs-core` the future -- antora based -- documentation?)

**Comment by Tschakki** [comment]
@fchavant There is no need to fix on `development` anymore, it's already outdated.
We should fix it in the new branches. 

And as long as we still use Kirby CMS, we can fix it directly in there as well, once it's reviewed.

**Comment by ghost** [comment]
@Tschakki OK, can you ensure this gets fixed in Kirby and close this PR then?
I'll create 2 more PRs for the `docs-core` and `docs-sdk` branches.

EDIT: see
- https://github.com/LiskHQ/lisk-docs/pull/350/ for Core
- https://github.com/LiskHQ/lisk-docs/pull/351 for SDK

**Comment by Tschakki** [comment]
Updated on kirby

**Pull request closed without merging**